src/snapshot/tests/test.rs

```diff
@@ -116,3 +116,115 @@ fn test_hardcoded_snapshot_deserialization() {
 
     assert_eq!(restored_struct, expected_struct);
 }
+
+#[test]
+fn test_invalid_format_version() {
+    #[rustfmt::skip]
+    let invalid_format_snap: &[u8] = &[
+        // This blob is consisted of the following: magic_id (8 bytes),
```

Comment:
Nit, same for next occurrences.

```suggestion
        // This blob consists of the following: magic_id (8 bytes),
```

Reply:
done